lib/auth/tun.go

```diff
@@ -992,7 +992,7 @@ func (c *TunClient) getClient() (client *ssh.Client, err error) {
 		if trace.IsAccessDenied(err) {
 			return nil, trace.Wrap(err)
 		}
-		log.Debugf("%v.getClient() throttle auth server %v: %v", c, authServer, err)
+		log.Errorf("%v.getClient() error while connecting to auth server %v: %v: throttling", c, authServer, err)
```

Comment:
@kontsevoy I see what you mean, I can see it being useful as well, because it informs us that we may skip over trying this auth server next time around: https://github.com/gravitational/teleport/blob/30de671dbab8ee874d019c63d89eea696e528b85/lib/auth/tun.go#L983-L985